source/features/render-semantic-release-commit-types.tsx

```diff
@@ -0,0 +1,112 @@
+import './render-semantic-release-commit-types.css';
+import React from 'react';
+import {$} from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager.js';
+import observe from '../helpers/selector-observer.js';
+import {commitTitleInLists} from '../github-helpers/selectors.js';
+import {getSemanticCommitAndScope, strip} from '../helpers/render-semantic-release-commit-types.js';
+
+const defaultCommitTypes = new Set([
+	'feat',
+	'fix',
+	'chore',
+	'docs',
+	'build',
+	'refactor',
+	'test',
+	'ci',
+	'perf',
+]);
+
+const defaultCommitTypesToLabelMapping = new Map([
+	['feat', 'Feature'],
+	['fix', 'Fix'],
+	['chore', 'Chore'],
+	['docs', 'Docs'],
+	['build', 'Build'],
+	['refactor', 'Refactor'],
+	['test', 'Test'],
+	['ci', 'CI'],
+	['perf', 'Performance'],
+]);
+
+function removeSemanticCommitTypeAndScopeFromCommitTitleElement(semanticCommitTitleElement: HTMLElement): void {
+	const [type, scope] = getSemanticCommitAndScope(semanticCommitTitleElement.textContent)!;
+
+	const semanticCommitTypeAndScope = scope ? `${type}(${scope}):` : `${type}:`;
+
+	strip(semanticCommitTitleElement, semanticCommitTypeAndScope);
```

Comment:
It’s safer to strip the exact match rather than having to reconstruct the string. So probably `getSemanticCommitAndScope` should return the full matched commit prefix. 

Also note that this is the 4th call of getSemanticCommitAndScope in this file. `removeSemanticCommitTypeAndScopeFromCommitTitleElement` should also be inlined because it’s just `strip(commitTitleElement, prefix)`